pkg/provider/branch/provider.go

```diff
@@ -407,7 +400,11 @@ func (b *branchENIProvider) CreateAndAnnotateResources(podNamespace string, podN
 func (b *branchENIProvider) DeleteBranchUsedByPods(nodeName string, UID string) (ctrl.Result, error) {
 	trunkENI, isPresent := b.getTrunkFromCache(nodeName)
 	if !isPresent {
-		return ctrl.Result{}, fmt.Errorf("failed to find trunk ENI on the node %s", nodeName)
+		// trunk cache is local map with lock. it shouldn't return not found if trunk exists
```

Comment:
nit: did you mean we shouldn't return error if trunk not found?

Reply:
This is not `error`
```
trunkENI, isPresent := b.getTrunkFromCache(nodeName)
```
The `isPresent` is a bool value to indicate if the key exists in the map.

Reply:
I meant the comment looks confusing, seems it should be something like `do not return not found error if trunk does not exist`?

Reply:
hmm, I see what you mean. Yeah I can make the comment better.